### PR TITLE
A tiny js world

### PR DESCRIPTION
**Pull request by vl-shevchenko**
### A tiny js world

[Demo](https://vl-shevchenko.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/vl-shevchenko/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by vl-shevchenko** [comment]
Hey!
I took into account all the tips from common mistakes when writing code.

**Comment by vl-shevchenko** [comment]
Hi!
In last commit I changed the method.

File: submissions/vl-shevchenko/js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+/* Print examples:
+   print('ABC');
+   print('<strong>ABC</strong>');
+   print('<strong>ABC</strong>', 'div');
+
+   print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
+   */
+let arr = [dog, cat, woman, man];
```
Comment (OleksiyRudenko) [inline submissions/vl-shevchenko/js-world/index.js:60]: From the checklist:
> - Give variables and functions meaningful names. Avoid generic names like `item`, `element`, `key`, `object`, `array` or their variations. Exception: helper functions that are specifically and intentionally designed to be multipurpose.

File: submissions/vl-shevchenko/js-world/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+arr.forEach(function ({ species, name, gender, legs, hands, saying }) {
+   print(`${species}; <strong>${name}</strong>; ${gender}; ${legs}; ${hands}; <em>${saying}</em>`);
```
Comment (OleksiyRudenko) [inline submissions/vl-shevchenko/js-world/index.js:62]: From the checklist:
> - [ ] Code is [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), which means that whenever you see a pattern in your code those should be eliminated as much as possible. Examples:
>    - `print(dog); print(cat); etc ... ` should be refactored employing `Array.forEach` as the least
>    - `` `${obj.legs}; ${obj.name}; etc...` `` (yes, strings are also code) must be refactored employing appropriate `Array` methods

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@vl-shevchenko good job. Although requires improvements.

**Comment by vl-shevchenko** [comment]
@OleksiyRudenko, I corrected all comments

**Review by OleksiyRudenko — APPROVED** [review]
@vl-shevchenko great job!

**Comment by OleksiyRudenko** [comment]
BTW there is a way to keep extra content even with current `.map` approach.
See [this](https://github.com/kottans/frontend-2022-homeworks/pull/391/files) for inspiration.

**Pull request merged**